src/cpp/RiderLink/Source/RiderLink/Public/RdEditorProtocol/RdEditorModel/RdEditorModel.cpp

```diff
@@ -35,17 +35,16 @@ namespace Jetbrains {
         //initializer
         void RdEditorModel::initialize()
         {
-            play_.optimize_nested = true;
             playMode_.optimize_nested = true;
             unrealLog_.async = true;
             onBlueprintAdded_.async = true;
-            serializationHash = 3261994287838629902L;
+            serializationHash = -1569663499128468552L;
```

Comment:
this looks like a generated value. it that ok that we commit generated code?

Reply:
I've commited generated part from rd-cpp because I had to add some manual changes to these files in order to build it with UE.
We'll be able to remove them from VCS when we'll have it working with UE out of the box